operator/charts/templates/webhook-server-cert-secret.yaml

```diff
@@ -1,11 +1,24 @@
+{{- if not .Values.config.server.webhooks.certManagement.certManagerEnabled }}
+---
 apiVersion: v1
 kind: Secret
 metadata:
-  name: grove-webhook-server-cert
+  name: {{ .Values.config.server.webhooks.certManagement.secretName }}
   namespace: {{ .Release.Namespace }}
   labels:
 {{- include "operator.server.secret.labels" . | nindent 4 }}
 type: kubernetes.io/tls
 data:
+  {{- if .Values.config.server.webhooks.certManagement.certFilesPath }}
+  # Certificate files provided: read from chart directory
+  tls.crt: {{ .Files.Get (printf "%s/server.crt" .Values.config.server.webhooks.certManagement.certFilesPath) | b64enc }}
```

Comment:
I don't understand why we're using .File.Get here?  

Cert Manager will bundle the following into a secret file:
```
tls.crt: The signed public certificate, often followed by the full intermediate certificate chain.

tls.key: The private key for the certificate.

ca.crt: The root CA certificate (this is common when using a private CA).

```

All that's needed is to mount this secret into the webhook deployment. Then the webhook service can use the certificate and private key. Note the ca.crt isn't used by the deployment itself. 

For those who aren't using Cert Manager, they can just create a similar secret however they want and everything will work.